frontend/src/components/common/header/index.css

```diff
@@ -90,20 +90,20 @@ a.dropdown-item {
 
 /* donate button on second navbar (green) */
 #donate-button {
+  margin-left: 5 0px;
   font-size: 20px;
   font-weight: bold;
   background-color: #fdd368;
   border-radius: 7px;
   border: none;
-  width: 140px;
-  height: 54px;
+  width: auto;
```

Comment:
This makes the button look really good on mobile but on Desktop it looks very condensed 

![image](https://user-images.githubusercontent.com/1637481/70485165-c7b1ae00-1ab3-11ea-9c13-da2cdaab1f47.png)

I think for this story we can keep the height/width and just change the `margin-left`